CHANGES.md

```diff
@@ -6,6 +6,17 @@ Version 0.24.1
 
 To be released.
 
+### Added APIs
+
+ -  Added `BlockChain<T>.IsCanonical` property.  [[#1672]]
```

Comment:
Patch releases cannot change APIs.  (That's why patch release notes do not have sections.)  Could you make it `internal` in this patch?

Reply:
Hmm... wasn't sure since it wasn't breaking. I suppose I can make it internal for now.

Reply:
Done.